docs/_sidebar.md

```diff
@@ -1,3 +1,6 @@
 <!-- markdownlint-disable -->
 
 * [Home :house_with_garden:](/)
+* Logic Apps
+  * [Controlling a single Azure Logic App](./logicapps/control-single-logicapp.md)
```

Comment:
This gives a 404

Reply:
It doesn't for me 😅.

Reply:
It seems like you don't get a 404 when clicking the link when on the homepage, from any other page I'm getting the 404 as well.

Reply:
This might help you @stijnmoreels - https://github.com/invictus-integration/docs-ifa/blob/master/_sidebar.md

Reply:
I removed the dot `.` at the beginning. That seems the only difference.